src/sentry/utils/sdk.py

```diff
@@ -78,6 +78,7 @@
     "sentry.dynamic_sampling.tasks.sliding_window": 0.2,
     "sentry.dynamic_sampling.tasks.sliding_window_org": 0.2,
     "sentry.dynamic_sampling.tasks.collect_orgs": 0.2,
+    "sentry.dynamic_sampling.tasks.custom_rule_notifications": 0.2,
```

Comment:
Just out of curiosity, why don't we sample these with a rate of 100%?

Reply:
0.2 is considered enough for tasks that run every 10 minutes ( so you get a bit more than 1 every hour).
I don't have a particular opinion on this, but I've seen that this is approximately what we do in these cases (see other similar tasks).